### PR TITLE
sklearn_to_pfa package

### PR DESCRIPTION
**Pull request by Marigold**
Python library for converting scikit-learn models to PFA.

### CODE REVIEW

**Comment by Marigold** [comment]
Created separate image python-mip-sklearn with sklearn_to_pfa library and scikit-learn in requirements. Merging this will allow removing duplicate code from algo-repository.

File: python-mip-sklearn/.bumpversion.cfg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[bumpversion]
+current_version = 0.2.5
```
Comment (ludovicc) [inline python-mip-sklearn/.bumpversion.cfg]: start with 0.1

File: python-mip-sklearn/README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+1. Extend a version of hbpmip/python-mip-sklearn image (see Dockerfile example below);
+2. Add `from mip_helper import mip_helper` in your Python script to import the library;
+3. Call `mip_helper.fetch_data()` to get the input data (formatted like described below);
+4. Call `mip_helper.save_results(pfa, error, shape)` to store the results;
+5. Ensure that your Dockerfile copies the needed files, sets up a valid entry-point
+and sets up a _FUNCTION_ environment variable containing the algorithm name (see Dockerfile example below).
+
+
+### Input format
```
Comment (ludovicc) [inline python-mip-sklearn/README.md]: Adapt the documentation to what this image provides

**Comment by ludovicc** [comment]
Merging anyway as latest version of histograms requires get_results with an argument. Please make further pull requests as required

**Comment by Marigold** [comment]
Haha, pushed like 30 seconds before you merged :).

**Pull request merged**